### PR TITLE
Add owners with all the maintainers

### PR DESCRIPTION
**Pull request by samj1912**

### CODE REVIEW

**Comment by samj1912** [comment]
cc @buildpacks/core-team @buildpacks/buildpack-authors-tooling-maintainers @buildpacks/implementation-maintainers @buildpacks/distribution-maintainers @buildpacks/platform-maintainers @buildpacks/learning-maintainers to verify their email addresses.

**Review by dfreilich — APPROVED** [review]
It would be great if we could have some docs, maybe in GOVERNANCE, which describes the steps to add someone as a maintainer (one of which would include adding the appropriate line to this, and perhaps requesting access to cncf list), so that this list doesn't get out of date

**Comment by hone** [comment]
Do we want to use vmware email addresses now?

@elbandito do you want to change to your personal email address?

**Review by hone — APPROVED** [review]

File: OWNERS.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Maintainers
+
+- Ben Hale, <bhale@pivotal.io> ([@nebhale](https://github.com/nebhale))
+- David Freilich, <freilich.david@gmail.com> ([@dfreilich](https://github.com/dfreilich))
+- Emily Casey, <ecasey@pivotal.io> ([@ekcasey](https://github.com/ekcasey))
+- Javier Romero, <j.romero.1214@gmail.com> ([@jromero](https://github.com/jromero))
```
Comment (jromero) [inline OWNERS.md:6]:
```suggestion
- Javier Romero, <root@jromero.codes> ([@jromero](https://github.com/jromero))
```

Can I use my cool domain name? I'm not paying the annual fees for nothing.

**Comment by samj1912** [comment]
> It would be great if we could have some docs, maybe in GOVERNANCE, which describes the steps to add someone as a maintainer (one of which would include adding the appropriate line to this, and perhaps requesting access to cncf list), so that this list doesn't get out of date

Yup, we need to consolidate the steps to add the maintainers somewhere. Currently we need this owners file so that CNCF can sync this file with service desk access and access to the buildpacks maintainers list.

> Do we want to use vmware email addresses now?

Don't know which ones to use. I copied the emails from slack.

**Comment by sclevine** [comment]
@ekcasey @nebhale @natalieparellano can you suggest alternative emails in this PR?

File: OWNERS.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Maintainers
+
+- Ben Hale, <bhale@pivotal.io> ([@nebhale](https://github.com/nebhale))
+- David Freilich, <freilich.david@gmail.com> ([@dfreilich](https://github.com/dfreilich))
+- Emily Casey, <ecasey@pivotal.io> ([@ekcasey](https://github.com/ekcasey))
+- Javier Romero, <root@jromero.codes> ([@jromero](https://github.com/jromero))
+- Jesse Brown, <jabrown85@gmail.com> ([@jabrown85](https://github.com/jabrown85))
+- Joe Kutner, <jpkutner@gmail.com> ([@jkutner](https://github.com/jkutner))
+- Natalie Arellano, <narellano@pivotal.io> ([@natalieparellano](https://github.com/natalieparellano))
```
Comment (natalieparellano) [inline OWNERS.md:9]:
```suggestion
- Natalie Arellano, <narellano@vmware.com> ([@natalieparellano](https://github.com/natalieparellano))
```

File: OWNERS.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Maintainers
+
+- Ben Hale, <bhale@pivotal.io> ([@nebhale](https://github.com/nebhale))
+- David Freilich, <freilich.david@gmail.com> ([@dfreilich](https://github.com/dfreilich))
+- Emily Casey, <ecasey@pivotal.io> ([@ekcasey](https://github.com/ekcasey))
```
Comment (ekcasey) [inline OWNERS.md:5]:
```suggestion
- Emily Casey, <ecasey@vmware.com> ([@ekcasey](https://github.com/ekcasey))
```

**Review by ekcasey — CHANGES_REQUESTED** [review]

**Comment by samj1912** [comment]
@sclevine moved to the OWNERS format similar to the one in spec.

File: OWNERS [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+David Freilich, AppsFlyer <freilich.david@gmail.com> (@dfreilich)
+Emily Casey, VMware <ecasey@vmware.com> (@ekcasey)
+Javier Romero, VMware <root@jromero.codes> (@jromero)
+Jesse Brown, Salesforce <jabrown85@gmail.com> (@jabrown85)
+Joe Kutner, Salesforce <jpkutner@gmail.com> (@jkutner)
+Natalie Arellano, VMware <narellano@vmware.com> (@natalieparellano)
+Sambhav Kothari, Bloomberg <sambhavs.email@gmail.com> (@samj1912)
+Stephen Levine, VMware <stephen@levine.sh> (@sclevine)
+Terence Lee, Salesforce <hone02@gmail.com> (@hone)
+Travis Longoria, Salesforce <tlongoria@heroku.com> (@elbandito)
```
Comment (elbandito) [inline OWNERS:11]: @samj1912 could you change my email to: `longoria.public@gmail.com`
Thanks!

**Review by jromero — APPROVED** [review]

**Review by jkutner — APPROVED** [review]

**Review by elbandito — APPROVED** [review]

**Review by hone — APPROVED** [review]

File: OWNERS [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Ben Hale, VMware <bhale@vmware.com> (@nebhale)
```
Comment (nebhale) [inline OWNERS:1]:
```suggestion
Ben Hale, VMware <nebhale@nebhale.com> (@nebhale)
```

**Review by nebhale — CHANGES_REQUESTED** [review]

**Review by sclevine — APPROVED** [review]

**Review by nebhale — APPROVED** [review]

File: OWNERS [hunk 6]
```diff
@@ -0,0 +1,11 @@
+Ben Hale, VMware <nebhale@nebhale.com> (@nebhale)
+David Freilich, AppsFlyer <freilich.david@gmail.com> (@dfreilich)
+Emily Casey, VMware <ecasey@vmware.com> (@ekcasey)
```
Comment (ekcasey) [inline OWNERS:3]:
```suggestion
Emily Casey, VMware <emilykimballcasey@gmail.com> (@ekcasey)
```

Sorry, changed my mind. I'd like to use my personal email.

**Review by ekcasey — APPROVED** [review]

**Comment by samj1912** [comment]
@dfreilich @natalieparellano @jabrown85 do your emails look good?

**Review by dfreilich — APPROVED** [review]

**Review by jabrown85 — APPROVED** [review]

**Pull request merged**